
Property changes on: .
___________________________________________________________________
Modified: svn:ignore
   - dist
*~
velocity.log
build.properties
.idea
*.iml
*.ipr
*.iws
.project
.classpath
.settings
prj.el
eclipse-build
maven-build

   + dist
*~
velocity.log
build.properties
.idea
*.iml
*.ipr
*.iws
.project
.classpath
.settings
prj.el
eclipse-build
idea-build
maven-build



Property changes on: lucene/benchmark/conf
___________________________________________________________________
Added: svn:ignore
   + *.iml



Property changes on: lucene/benchmark/src
___________________________________________________________________
Added: svn:ignore
   + *.iml


Index: dev-tools/idea/parent.iml
===================================================================
--- dev-tools/idea/parent.iml	(revision 1384534)
+++ dev-tools/idea/parent.iml	(working copy)
@@ -6,6 +6,8 @@
       <excludeFolder url="file://$MODULE_DIR$/dist" />
       <excludeFolder url="file://$MODULE_DIR$/lucene/build" />
       <excludeFolder url="file://$MODULE_DIR$/lucene/dist" />
+      <excludeFolder url="file://$MODULE_DIR$/lucene/benchmark/temp" />
+      <excludeFolder url="file://$MODULE_DIR$/lucene/benchmark/work" />
       <excludeFolder url="file://$MODULE_DIR$/solr/build" />
       <excludeFolder url="file://$MODULE_DIR$/solr/dist" />
       <excludeFolder url="file://$MODULE_DIR$/solr/package" />
Index: dev-tools/idea/lucene/demo/demo.iml
===================================================================
--- dev-tools/idea/lucene/demo/demo.iml	(revision 1384534)
+++ dev-tools/idea/lucene/demo/demo.iml	(working copy)
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <module type="JAVA_MODULE" version="4">
   <component name="NewModuleRootManager" inherit-compiler-output="false">
-    <output url="file://$MODULE_DIR$/../build/demo/classes/java" />
-    <output-test url="file://$MODULE_DIR$/../build/demo/classes/test" />
+    <output url="file://$MODULE_DIR$/../../idea-build/lucene/demo/classes/java" />
+    <output-test url="file://$MODULE_DIR$/../../idea-build/lucene/demo/classes/test" />
     <exclude-output />
     <content url="file://$MODULE_DIR$">
       <sourceFolder url="file://$MODULE_DIR$/src/java" isTestSource="false" />
Index: dev-tools/idea/lucene/core/src/test/lucene-core-tests.iml
===================================================================
--- dev-tools/idea/lucene/core/src/test/lucene-core-tests.iml	(revision 1384534)
+++ dev-tools/idea/lucene/core/src/test/lucene-core-tests.iml	(working copy)
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <module type="JAVA_MODULE" version="4">
   <component name="NewModuleRootManager" inherit-compiler-output="false">
-    <output url="file://$MODULE_DIR$/../../../build/core/classes/java" />
-    <output-test url="file://$MODULE_DIR$/../../../build/core/classes/test" />
+    <output url="file://$MODULE_DIR$/../../../../idea-build/lucene/core/classes/java" />
+    <output-test url="file://$MODULE_DIR$/../../../../idea-build/lucene/core/classes/test" />
     <exclude-output />
     <content url="file://$MODULE_DIR$">
       <sourceFolder url="file://$MODULE_DIR$" isTestSource="true" />
Index: dev-tools/idea/lucene/core/src/java/lucene-core.iml
===================================================================
--- dev-tools/idea/lucene/core/src/java/lucene-core.iml	(revision 1384534)
+++ dev-tools/idea/lucene/core/src/java/lucene-core.iml	(working copy)
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <module type="JAVA_MODULE" version="4">
   <component name="NewModuleRootManager" inherit-compiler-output="false">
-    <output url="file://$MODULE_DIR$/../../../build/core/classes/java" />
-    <output-test url="file://$MODULE_DIR$/../../../build/core/classes/test" />
+    <output url="file://$MODULE_DIR$/../../../../idea-build/lucene/core/classes/java" />
+    <output-test url="file://$MODULE_DIR$/../../../../idea-build/lucene/core/classes/test" />
     <exclude-output />
     <content url="file://$MODULE_DIR$">
       <sourceFolder url="file://$MODULE_DIR$" isTestSource="false" />
Index: dev-tools/idea/lucene/core/src/resources/lucene-core-resources.iml
===================================================================
--- dev-tools/idea/lucene/core/src/resources/lucene-core-resources.iml	(revision 1384534)
+++ dev-tools/idea/lucene/core/src/resources/lucene-core-resources.iml	(working copy)
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <module type="JAVA_MODULE" version="4">
   <component name="NewModuleRootManager" inherit-compiler-output="false">
-    <output url="file://$MODULE_DIR$/../../../build/core/classes/java" />
-    <output-test url="file://$MODULE_DIR$/../../../build/core/classes/test" />
+    <output url="file://$MODULE_DIR$/../../../../idea-build/lucene/core/classes/java" />
+    <output-test url="file://$MODULE_DIR$/../../../../idea-build/lucene/core/classes/test" />
     <exclude-output />
     <content url="file://$MODULE_DIR$">
       <sourceFolder url="file://$MODULE_DIR$" isTestSource="false" />
Index: dev-tools/idea/lucene/grouping/grouping.iml
===================================================================
--- dev-tools/idea/lucene/grouping/grouping.iml	(revision 1384534)
+++ dev-tools/idea/lucene/grouping/grouping.iml	(working copy)
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <module type="JAVA_MODULE" version="4">
   <component name="NewModuleRootManager" inherit-compiler-output="false">
-    <output url="file://$MODULE_DIR$/../build/grouping/classes/java" />
-    <output-test url="file://$MODULE_DIR$/../build/grouping/classes/test" />
+    <output url="file://$MODULE_DIR$/../../idea-build/lucene/grouping/classes/java" />
+    <output-test url="file://$MODULE_DIR$/../../idea-build/lucene/grouping/classes/test" />
     <exclude-output />
     <content url="file://$MODULE_DIR$">
       <sourceFolder url="file://$MODULE_DIR$/src/java" isTestSource="false" />
Index: dev-tools/idea/lucene/benchmark/conf/benchmark-conf.iml
===================================================================
--- dev-tools/idea/lucene/benchmark/conf/benchmark-conf.iml	(revision 0)
+++ dev-tools/idea/lucene/benchmark/conf/benchmark-conf.iml	(revision 0)
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="JAVA_MODULE" version="4">
+  <component name="NewModuleRootManager" inherit-compiler-output="false">
+    <output url="file://$MODULE_DIR$/../../../idea-build/lucene/benchmark/classes/java/conf" />
+    <output-test url="file://$MODULE_DIR$/../../../idea-build/lucene/benchmark/classes/test/conf" />
+    <exclude-output />
+    <content url="file://$MODULE_DIR$">
+      <sourceFolder url="file://$MODULE_DIR$" isTestSource="true" />
+    </content>
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+  </component>
+</module>
Index: dev-tools/idea/lucene/benchmark/src/benchmark.iml
===================================================================
--- dev-tools/idea/lucene/benchmark/src/benchmark.iml	(revision 0)
+++ dev-tools/idea/lucene/benchmark/src/benchmark.iml	(working copy)
@@ -1,30 +1,29 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <module type="JAVA_MODULE" version="4">
   <component name="NewModuleRootManager" inherit-compiler-output="false">
-    <output url="file://$MODULE_DIR$/../build/benchmark/classes/java" />
-    <output-test url="file://$MODULE_DIR$/../build/benchmark/classes/test" />
+    <output url="file://$MODULE_DIR$/../../../idea-build/lucene/benchmark/classes/java" />
+    <output-test url="file://$MODULE_DIR$/../../../idea-build/lucene/benchmark/classes/test" />
     <exclude-output />
     <content url="file://$MODULE_DIR$">
-      <sourceFolder url="file://$MODULE_DIR$/src/java" isTestSource="false" />
-      <sourceFolder url="file://$MODULE_DIR$/src/test" isTestSource="true" />
-      <excludeFolder url="file://$MODULE_DIR$/temp" />
-      <excludeFolder url="file://$MODULE_DIR$/work" />
+      <sourceFolder url="file://$MODULE_DIR$/java" isTestSource="false" />
+      <sourceFolder url="file://$MODULE_DIR$/test" isTestSource="true" />
     </content>
     <orderEntry type="inheritedJdk" />
     <orderEntry type="sourceFolder" forTests="false" />
     <orderEntry type="module-library">
       <library>
         <CLASSES>
-          <root url="file://$MODULE_DIR$/lib" />
+          <root url="file://$MODULE_DIR$/../lib" />
         </CLASSES>
         <JAVADOC />
         <SOURCES />
-        <jarDirectory url="file://$MODULE_DIR$/lib" recursive="false" />
+        <jarDirectory url="file://$MODULE_DIR$/../lib" recursive="false" />
       </library>
     </orderEntry>
     <orderEntry type="library" name="ICU library" level="project" />
     <orderEntry type="library" scope="TEST" name="JUnit" level="project" />
     <orderEntry type="module" scope="TEST" module-name="lucene-test-framework" />
+    <orderEntry type="module" scope="TEST" module-name="benchmark-conf" />
     <orderEntry type="module" module-name="facet" />
     <orderEntry type="module" module-name="highlighter" />
     <orderEntry type="module" module-name="icu" />

Property changes on: dev-tools/idea/lucene/benchmark/src/benchmark.iml
___________________________________________________________________
Added: svn:eol-style
   + native

Index: dev-tools/idea/lucene/benchmark/benchmark.iml
===================================================================
--- dev-tools/idea/lucene/benchmark/benchmark.iml	(revision 1384534)
+++ dev-tools/idea/lucene/benchmark/benchmark.iml	(working copy)
@@ -1,37 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<module type="JAVA_MODULE" version="4">
-  <component name="NewModuleRootManager" inherit-compiler-output="false">
-    <output url="file://$MODULE_DIR$/../build/benchmark/classes/java" />
-    <output-test url="file://$MODULE_DIR$/../build/benchmark/classes/test" />
-    <exclude-output />
-    <content url="file://$MODULE_DIR$">
-      <sourceFolder url="file://$MODULE_DIR$/src/java" isTestSource="false" />
-      <sourceFolder url="file://$MODULE_DIR$/src/test" isTestSource="true" />
-      <excludeFolder url="file://$MODULE_DIR$/temp" />
-      <excludeFolder url="file://$MODULE_DIR$/work" />
-    </content>
-    <orderEntry type="inheritedJdk" />
-    <orderEntry type="sourceFolder" forTests="false" />
-    <orderEntry type="module-library">
-      <library>
-        <CLASSES>
-          <root url="file://$MODULE_DIR$/lib" />
-        </CLASSES>
-        <JAVADOC />
-        <SOURCES />
-        <jarDirectory url="file://$MODULE_DIR$/lib" recursive="false" />
-      </library>
-    </orderEntry>
-    <orderEntry type="library" name="ICU library" level="project" />
-    <orderEntry type="library" scope="TEST" name="JUnit" level="project" />
-    <orderEntry type="module" scope="TEST" module-name="lucene-test-framework" />
-    <orderEntry type="module" module-name="facet" />
-    <orderEntry type="module" module-name="highlighter" />
-    <orderEntry type="module" module-name="icu" />
-    <orderEntry type="module" module-name="misc" />
-    <orderEntry type="module" module-name="memory" />
-    <orderEntry type="module" module-name="analysis-common" />
-    <orderEntry type="module" module-name="lucene-core" />
-    <orderEntry type="module" module-name="queryparser" />
-  </component>
-</module>
Index: dev-tools/idea/lucene/misc/misc.iml
===================================================================
--- dev-tools/idea/lucene/misc/misc.iml	(revision 1384534)
+++ dev-tools/idea/lucene/misc/misc.iml	(working copy)
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <module type="JAVA_MODULE" version="4">
   <component name="NewModuleRootManager" inherit-compiler-output="false">
-    <output url="file://$MODULE_DIR$/../build/misc/classes/java" />
-    <output-test url="file://$MODULE_DIR$/../build/misc/classes/test" />
+    <output url="file://$MODULE_DIR$/../../idea-build/lucene/misc/classes/java" />
+    <output-test url="file://$MODULE_DIR$/../../idea-build/lucene/misc/classes/test" />
     <exclude-output />
     <content url="file://$MODULE_DIR$">
       <sourceFolder url="file://$MODULE_DIR$/src/test" isTestSource="true" />
Index: dev-tools/idea/lucene/classification/classification.iml
===================================================================
--- dev-tools/idea/lucene/classification/classification.iml	(revision 1384534)
+++ dev-tools/idea/lucene/classification/classification.iml	(working copy)
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <module type="JAVA_MODULE" version="4">
   <component name="NewModuleRootManager" inherit-compiler-output="false">
-    <output url="file://$MODULE_DIR$/../build/classification/classes/java" />
-    <output-test url="file://$MODULE_DIR$/../build/classification/classes/test" />
+    <output url="file://$MODULE_DIR$/../../idea-build/lucene/classification/classes/java" />
+    <output-test url="file://$MODULE_DIR$/../../idea-build/lucene/classification/classes/test" />
     <exclude-output />
     <content url="file://$MODULE_DIR$">
       <sourceFolder url="file://$MODULE_DIR$/src/java" isTestSource="false" />
Index: dev-tools/idea/lucene/spatial/spatial.iml
===================================================================
--- dev-tools/idea/lucene/spatial/spatial.iml	(revision 1384534)
+++ dev-tools/idea/lucene/spatial/spatial.iml	(working copy)
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <module type="JAVA_MODULE" version="4">
   <component name="NewModuleRootManager" inherit-compiler-output="false">
-    <output url="file://$MODULE_DIR$/../build/spatial/classes/java" />
-    <output-test url="file://$MODULE_DIR$/../build/spatial/classes/test" />
+    <output url="file://$MODULE_DIR$/../../idea-build/lucene/spatial/classes/java" />
+    <output-test url="file://$MODULE_DIR$/../../idea-build/lucene/spatial/classes/test" />
     <exclude-output />
     <content url="file://$MODULE_DIR$">
       <sourceFolder url="file://$MODULE_DIR$/src/java" isTestSource="false" />
Index: dev-tools/idea/lucene/highlighter/highlighter.iml
===================================================================
--- dev-tools/idea/lucene/highlighter/highlighter.iml	(revision 1384534)
+++ dev-tools/idea/lucene/highlighter/highlighter.iml	(working copy)
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <module type="JAVA_MODULE" version="4">
   <component name="NewModuleRootManager" inherit-compiler-output="false">
-    <output url="file://$MODULE_DIR$/../build/highlighter/classes/java" />
-    <output-test url="file://$MODULE_DIR$/../build/highlighter/classes/test" />
+    <output url="file://$MODULE_DIR$/../../idea-build/lucene/highlighter/classes/java" />
+    <output-test url="file://$MODULE_DIR$/../../idea-build/lucene/highlighter/classes/test" />
     <exclude-output />
     <content url="file://$MODULE_DIR$">
       <sourceFolder url="file://$MODULE_DIR$/src/java" isTestSource="false" />
Index: dev-tools/idea/lucene/sandbox/sandbox.iml
===================================================================
--- dev-tools/idea/lucene/sandbox/sandbox.iml	(revision 1384534)
+++ dev-tools/idea/lucene/sandbox/sandbox.iml	(working copy)
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <module type="JAVA_MODULE" version="4">
   <component name="NewModuleRootManager" inherit-compiler-output="false">
-    <output url="file://$MODULE_DIR$/../build/sandbox/classes/java" />
-    <output-test url="file://$MODULE_DIR$/../build/sandbox/classes/test" />
+    <output url="file://$MODULE_DIR$/../../idea-build/lucene/sandbox/classes/java" />
+    <output-test url="file://$MODULE_DIR$/../../idea-build/lucene/sandbox/classes/test" />
     <exclude-output />
     <content url="file://$MODULE_DIR$">
       <sourceFolder url="file://$MODULE_DIR$/src/java" isTestSource="false" />
Index: dev-tools/idea/lucene/join/join.iml
===================================================================
--- dev-tools/idea/lucene/join/join.iml	(revision 1384534)
+++ dev-tools/idea/lucene/join/join.iml	(working copy)
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <module type="JAVA_MODULE" version="4">
   <component name="NewModuleRootManager" inherit-compiler-output="false">
-    <output url="file://$MODULE_DIR$/../build/join/classes/java" />
-    <output-test url="file://$MODULE_DIR$/../build/join/classes/test" />
+    <output url="file://$MODULE_DIR$/../../idea-build/lucene/join/classes/java" />
+    <output-test url="file://$MODULE_DIR$/../../idea-build/lucene/join/classes/test" />
     <exclude-output />
     <content url="file://$MODULE_DIR$">
       <sourceFolder url="file://$MODULE_DIR$/src/java" isTestSource="false" />
Index: dev-tools/idea/lucene/tools/tools.iml
===================================================================
--- dev-tools/idea/lucene/tools/tools.iml	(revision 1384534)
+++ dev-tools/idea/lucene/tools/tools.iml	(working copy)
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <module type="JAVA_MODULE" version="4">
   <component name="NewModuleRootManager" inherit-compiler-output="false">
-    <output url="file://$MODULE_DIR$/../build/tools/classes/java" />
-    <output-test url="file://$MODULE_DIR$/../build/tools/classes/test" />
+    <output url="file://$MODULE_DIR$/../../idea-build/lucene/tools/classes/java" />
+    <output-test url="file://$MODULE_DIR$/../../idea-build/lucene/tools/classes/test" />
     <exclude-output />
     <content url="file://$MODULE_DIR$">
       <sourceFolder url="file://$MODULE_DIR$/src/java" isTestSource="false" />
Index: dev-tools/idea/lucene/codecs/src/resources/codecs-resources.iml
===================================================================
--- dev-tools/idea/lucene/codecs/src/resources/codecs-resources.iml	(revision 1384534)
+++ dev-tools/idea/lucene/codecs/src/resources/codecs-resources.iml	(working copy)
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <module type="JAVA_MODULE" version="4">
   <component name="NewModuleRootManager" inherit-compiler-output="false">
-    <output url="file://$MODULE_DIR$/../../../build/codecs/classes/java" />
-    <output-test url="file://$MODULE_DIR$/../../../build/codecs/classes/test" />
+    <output url="file://$MODULE_DIR$/../../../../idea-build/lucene/codecs/classes/java" />
+    <output-test url="file://$MODULE_DIR$/../../../../idea-build/lucene/codecs/classes/test" />
     <exclude-output />
     <content url="file://$MODULE_DIR$">
       <sourceFolder url="file://$MODULE_DIR$" isTestSource="false" />
Index: dev-tools/idea/lucene/codecs/src/test/codecs-tests.iml
===================================================================
--- dev-tools/idea/lucene/codecs/src/test/codecs-tests.iml	(revision 1384534)
+++ dev-tools/idea/lucene/codecs/src/test/codecs-tests.iml	(working copy)
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <module type="JAVA_MODULE" version="4">
   <component name="NewModuleRootManager" inherit-compiler-output="false">
-    <output url="file://$MODULE_DIR$/../../../build/codecs/classes/java" />
-    <output-test url="file://$MODULE_DIR$/../../../build/codecs/classes/test" />
+    <output url="file://$MODULE_DIR$/../../../../idea-build/lucene/codecs/classes/java" />
+    <output-test url="file://$MODULE_DIR$/../../../../idea-build/lucene/codecs/classes/test" />
     <exclude-output />
     <content url="file://$MODULE_DIR$">
       <sourceFolder url="file://$MODULE_DIR$" isTestSource="true" />
Index: dev-tools/idea/lucene/codecs/src/java/codecs.iml
===================================================================
--- dev-tools/idea/lucene/codecs/src/java/codecs.iml	(revision 1384534)
+++ dev-tools/idea/lucene/codecs/src/java/codecs.iml	(working copy)
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <module type="JAVA_MODULE" version="4">
   <component name="NewModuleRootManager" inherit-compiler-output="false">
-    <output url="file://$MODULE_DIR$/../../../build/codecs/classes/java" />
-    <output-test url="file://$MODULE_DIR$/../../../build/codecs/classes/test" />
+    <output url="file://$MODULE_DIR$/../../../../idea-build/lucene/codecs/classes/java" />
+    <output-test url="file://$MODULE_DIR$/../../../../idea-build/lucene/codecs/classes/test" />
     <exclude-output />
     <content url="file://$MODULE_DIR$">
       <sourceFolder url="file://$MODULE_DIR$" isTestSource="false" />
Index: dev-tools/idea/lucene/memory/memory.iml
===================================================================
--- dev-tools/idea/lucene/memory/memory.iml	(revision 1384534)
+++ dev-tools/idea/lucene/memory/memory.iml	(working copy)
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <module type="JAVA_MODULE" version="4">
   <component name="NewModuleRootManager" inherit-compiler-output="false">
-    <output url="file://$MODULE_DIR$/../build/memory/classes/java" />
-    <output-test url="file://$MODULE_DIR$/../build/memory/classes/test" />
+    <output url="file://$MODULE_DIR$/../../idea-build/lucene/memory/classes/java" />
+    <output-test url="file://$MODULE_DIR$/../../idea-build/lucene/memory/classes/test" />
     <exclude-output />
     <content url="file://$MODULE_DIR$">
       <sourceFolder url="file://$MODULE_DIR$/src/java" isTestSource="false" />
Index: dev-tools/idea/lucene/test-framework/lucene-test-framework.iml
===================================================================
--- dev-tools/idea/lucene/test-framework/lucene-test-framework.iml	(revision 1384534)
+++ dev-tools/idea/lucene/test-framework/lucene-test-framework.iml	(working copy)
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <module type="JAVA_MODULE" version="4">
   <component name="NewModuleRootManager" inherit-compiler-output="false">
-    <output url="file://$MODULE_DIR$/../build/test-framework/classes/java" />
-    <output-test url="file://$MODULE_DIR$/../build/test-framework/classes/test" />
+    <output url="file://$MODULE_DIR$/../../idea-build/lucene/test-framework/classes/java" />
+    <output-test url="file://$MODULE_DIR$/../../idea-build/lucene/test-framework/classes/test" />
     <exclude-output />
     <content url="file://$MODULE_DIR$">
       <sourceFolder url="file://$MODULE_DIR$/src/resources" isTestSource="false" />
Index: dev-tools/idea/lucene/queries/queries.iml
===================================================================
--- dev-tools/idea/lucene/queries/queries.iml	(revision 1384534)
+++ dev-tools/idea/lucene/queries/queries.iml	(working copy)
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <module type="JAVA_MODULE" version="4">
   <component name="NewModuleRootManager" inherit-compiler-output="false">
-    <output url="file://$MODULE_DIR$/../build/queries/classes/java" />
-    <output-test url="file://$MODULE_DIR$/../build/queries/classes/test" />
+    <output url="file://$MODULE_DIR$/../../idea-build/lucene/queries/classes/java" />
+    <output-test url="file://$MODULE_DIR$/../../idea-build/lucene/queries/classes/test" />
     <exclude-output />
     <content url="file://$MODULE_DIR$">
       <sourceFolder url="file://$MODULE_DIR$/src/java" isTestSource="false" />
Index: dev-tools/idea/lucene/suggest/suggest.iml
===================================================================
--- dev-tools/idea/lucene/suggest/suggest.iml	(revision 1384534)
+++ dev-tools/idea/lucene/suggest/suggest.iml	(working copy)
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <module type="JAVA_MODULE" version="4">
   <component name="NewModuleRootManager" inherit-compiler-output="false">
-    <output url="file://$MODULE_DIR$/../build/suggest/classes/java" />
-    <output-test url="file://$MODULE_DIR$/../build/suggest/classes/test" />
+    <output url="file://$MODULE_DIR$/../../idea-build/lucene/suggest/classes/java" />
+    <output-test url="file://$MODULE_DIR$/../../idea-build/lucene/suggest/classes/test" />
     <exclude-output />
     <content url="file://$MODULE_DIR$">
       <sourceFolder url="file://$MODULE_DIR$/src/java" isTestSource="false" />
Index: dev-tools/idea/lucene/facet/facet.iml
===================================================================
--- dev-tools/idea/lucene/facet/facet.iml	(revision 1384534)
+++ dev-tools/idea/lucene/facet/facet.iml	(working copy)
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <module type="JAVA_MODULE" version="4">
   <component name="NewModuleRootManager" inherit-compiler-output="false">
-    <output url="file://$MODULE_DIR$/../build/facet/classes/java" />
-    <output-test url="file://$MODULE_DIR$/../build/facet/classes/test" />
+    <output url="file://$MODULE_DIR$/../../idea-build/lucene/facet/classes/java" />
+    <output-test url="file://$MODULE_DIR$/../../idea-build/lucene/facet/classes/test" />
     <exclude-output />
     <content url="file://$MODULE_DIR$">
       <sourceFolder url="file://$MODULE_DIR$/src/java" isTestSource="false" />
Index: dev-tools/idea/lucene/queryparser/queryparser.iml
===================================================================
--- dev-tools/idea/lucene/queryparser/queryparser.iml	(revision 1384534)
+++ dev-tools/idea/lucene/queryparser/queryparser.iml	(working copy)
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <module type="JAVA_MODULE" version="4">
   <component name="NewModuleRootManager" inherit-compiler-output="false">
-    <output url="file://$MODULE_DIR$/../build/queryparser/classes/java" />
-    <output-test url="file://$MODULE_DIR$/../build/queryparser/classes/test" />
+    <output url="file://$MODULE_DIR$/../../idea-build/lucene/queryparser/classes/java" />
+    <output-test url="file://$MODULE_DIR$/../../idea-build/lucene/queryparser/classes/test" />
     <exclude-output />
     <content url="file://$MODULE_DIR$">
       <sourceFolder url="file://$MODULE_DIR$/src/java" isTestSource="false" />
Index: dev-tools/idea/lucene/analysis/uima/analysis-uima.iml
===================================================================
--- dev-tools/idea/lucene/analysis/uima/analysis-uima.iml	(revision 1384534)
+++ dev-tools/idea/lucene/analysis/uima/analysis-uima.iml	(working copy)
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <module type="JAVA_MODULE" version="4">
   <component name="NewModuleRootManager" inherit-compiler-output="false">
-    <output url="file://$MODULE_DIR$/../../build/analysis/analyzers-uima/classes/java" />
-    <output-test url="file://$MODULE_DIR$/../../build/analysis/analyzers-uima/classes/test" />
+    <output url="file://$MODULE_DIR$/../../../idea-build/lucene/analysis/uima/classes/java" />
+    <output-test url="file://$MODULE_DIR$/../../../idea-build/lucene/analysis/uima/classes/test" />
     <exclude-output />
     <content url="file://$MODULE_DIR$">
       <sourceFolder url="file://$MODULE_DIR$/src/java" isTestSource="false" />
Index: dev-tools/idea/lucene/analysis/kuromoji/kuromoji.iml
===================================================================
--- dev-tools/idea/lucene/analysis/kuromoji/kuromoji.iml	(revision 1384534)
+++ dev-tools/idea/lucene/analysis/kuromoji/kuromoji.iml	(working copy)
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <module type="JAVA_MODULE" version="4">
   <component name="NewModuleRootManager" inherit-compiler-output="false">
-    <output url="file://$MODULE_DIR$/../../build/analysis/analyzers-kuromoji-idea/classes/java" />
-    <output-test url="file://$MODULE_DIR$/../../build/analysis/analyzers-kuromoji-idea/classes/test" />
+    <output url="file://$MODULE_DIR$/../../../idea-build/lucene/analysis/kuromoji/classes/java" />
+    <output-test url="file://$MODULE_DIR$/../../../idea-build/lucene/analysis/kuromoji/classes/test" />
     <exclude-output />
     <content url="file://$MODULE_DIR$">
       <sourceFolder url="file://$MODULE_DIR$/src/java" isTestSource="false" />
Index: dev-tools/idea/lucene/analysis/icu/icu.iml
===================================================================
--- dev-tools/idea/lucene/analysis/icu/icu.iml	(revision 1384534)
+++ dev-tools/idea/lucene/analysis/icu/icu.iml	(working copy)
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <module type="JAVA_MODULE" version="4">
   <component name="NewModuleRootManager" inherit-compiler-output="false">
-    <output url="file://$MODULE_DIR$/../../build/analysis/analyzers-icu/classes/java" />
-    <output-test url="file://$MODULE_DIR$/../../build/analysis/analyzers-icu/classes/test" />
+    <output url="file://$MODULE_DIR$/../../../idea-build/lucene/analysis/icu/classes/java" />
+    <output-test url="file://$MODULE_DIR$/../../../idea-build/lucene/analysis/icu/classes/test" />
     <exclude-output />
     <content url="file://$MODULE_DIR$">
       <sourceFolder url="file://$MODULE_DIR$/src/java" isTestSource="false" />
Index: dev-tools/idea/lucene/analysis/smartcn/smartcn.iml
===================================================================
--- dev-tools/idea/lucene/analysis/smartcn/smartcn.iml	(revision 1384534)
+++ dev-tools/idea/lucene/analysis/smartcn/smartcn.iml	(working copy)
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <module type="JAVA_MODULE" version="4">
   <component name="NewModuleRootManager" inherit-compiler-output="false">
-    <output url="file://$MODULE_DIR$/../../build/analysis/analyzers-smartcn/classes/java" />
-    <output-test url="file://$MODULE_DIR$/../../build/analysis/analyzers-smartcn/classes/test" />
+    <output url="file://$MODULE_DIR$/../../../idea-build/lucene/analysis/smartcn/classes/java" />
+    <output-test url="file://$MODULE_DIR$/../../../idea-build/lucene/analysis/smartcn/classes/test" />
     <exclude-output />
     <content url="file://$MODULE_DIR$">
       <sourceFolder url="file://$MODULE_DIR$/src/java" isTestSource="false" />
Index: dev-tools/idea/lucene/analysis/common/analysis-common.iml
===================================================================
--- dev-tools/idea/lucene/analysis/common/analysis-common.iml	(revision 1384534)
+++ dev-tools/idea/lucene/analysis/common/analysis-common.iml	(working copy)
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <module type="JAVA_MODULE" version="4">
   <component name="NewModuleRootManager" inherit-compiler-output="false">
-    <output url="file://$MODULE_DIR$/../../build/analysis/analyzers-common/classes/java" />
-    <output-test url="file://$MODULE_DIR$/../../build/analysis/analyzers-common/classes/test" />
+    <output url="file://$MODULE_DIR$/../../../idea-build/lucene/analysis/common/classes/java" />
+    <output-test url="file://$MODULE_DIR$/../../../idea-build/lucene/analysis/common/classes/test" />
     <exclude-output />
     <content url="file://$MODULE_DIR$">
       <sourceFolder url="file://$MODULE_DIR$/src/java" isTestSource="false" />
Index: dev-tools/idea/lucene/analysis/morfologik/morfologik.iml
===================================================================
--- dev-tools/idea/lucene/analysis/morfologik/morfologik.iml	(revision 1384534)
+++ dev-tools/idea/lucene/analysis/morfologik/morfologik.iml	(working copy)
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <module type="JAVA_MODULE" version="4">
   <component name="NewModuleRootManager" inherit-compiler-output="false">
-    <output url="file://$MODULE_DIR$/../../build/analysis/analyzers-morfologik/classes/java" />
-    <output-test url="file://$MODULE_DIR$/../../build/analysis/analyzers-morfologik/classes/test" />
+    <output url="file://$MODULE_DIR$/../../../idea-build/lucene/analysis/morfologik/classes/java" />
+    <output-test url="file://$MODULE_DIR$/../../../idea-build/lucene/analysis/morfologik/classes/test" />
     <exclude-output />
     <content url="file://$MODULE_DIR$">
       <sourceFolder url="file://$MODULE_DIR$/src/java" isTestSource="false" />
Index: dev-tools/idea/lucene/analysis/phonetic/phonetic.iml
===================================================================
--- dev-tools/idea/lucene/analysis/phonetic/phonetic.iml	(revision 1384534)
+++ dev-tools/idea/lucene/analysis/phonetic/phonetic.iml	(working copy)
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <module type="JAVA_MODULE" version="4">
   <component name="NewModuleRootManager" inherit-compiler-output="false">
-    <output url="file://$MODULE_DIR$/../../build/analysis/analyzers-phonetic/classes/java" />
-    <output-test url="file://$MODULE_DIR$/../../build/analysis/analyzers-phonetic/classes/test" />
+    <output url="file://$MODULE_DIR$/../../../idea-build/lucene/analysis/phonetic/classes/java" />
+    <output-test url="file://$MODULE_DIR$/../../../idea-build/lucene/analysis/phonetic/classes/test" />
     <exclude-output />
     <content url="file://$MODULE_DIR$">
       <sourceFolder url="file://$MODULE_DIR$/src/java" isTestSource="false" />
Index: dev-tools/idea/lucene/analysis/stempel/stempel.iml
===================================================================
--- dev-tools/idea/lucene/analysis/stempel/stempel.iml	(revision 1384534)
+++ dev-tools/idea/lucene/analysis/stempel/stempel.iml	(working copy)
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <module type="JAVA_MODULE" version="4">
   <component name="NewModuleRootManager" inherit-compiler-output="false">
-    <output url="file://$MODULE_DIR$/../../build/analysis/analyzers-stempel/classes/java" />
-    <output-test url="file://$MODULE_DIR$/../../build/analysis/analyzers-stempel/classes/test" />
+    <output url="file://$MODULE_DIR$/../../../idea-build/lucene/analysis/stempel/classes/java" />
+    <output-test url="file://$MODULE_DIR$/../../../idea-build/lucene/analysis/stempel/classes/test" />
     <exclude-output />
     <content url="file://$MODULE_DIR$">
       <sourceFolder url="file://$MODULE_DIR$/src/test" isTestSource="true" />
Index: dev-tools/idea/solr/core/src/test/solr-core-tests.iml
===================================================================
--- dev-tools/idea/solr/core/src/test/solr-core-tests.iml	(revision 1384534)
+++ dev-tools/idea/solr/core/src/test/solr-core-tests.iml	(working copy)
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <module type="JAVA_MODULE" version="4">
   <component name="NewModuleRootManager" inherit-compiler-output="false">
-    <output url="file://$MODULE_DIR$/../../../build/solr-core/classes/java" />
-    <output-test url="file://$MODULE_DIR$/../../../build/solr-core/classes/test" />
+    <output url="file://$MODULE_DIR$/../../../../idea-build/solr/solr-core/classes/java" />
+    <output-test url="file://$MODULE_DIR$/../../../../idea-build/solr/solr-core/classes/test" />
     <exclude-output />
     <content url="file://$MODULE_DIR$">
       <sourceFolder url="file://$MODULE_DIR$" isTestSource="true" />
Index: dev-tools/idea/solr/core/src/test-files/solr-core-test-files.iml
===================================================================
--- dev-tools/idea/solr/core/src/test-files/solr-core-test-files.iml	(revision 1384534)
+++ dev-tools/idea/solr/core/src/test-files/solr-core-test-files.iml	(working copy)
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <module type="JAVA_MODULE" version="4">
   <component name="NewModuleRootManager" inherit-compiler-output="false">
-    <output url="file://$MODULE_DIR$/../../../build/solr-core/classes/java" />
-    <output-test url="file://$MODULE_DIR$/../../../build/solr-core/classes/test" />
+    <output url="file://$MODULE_DIR$/../../../../idea-build/solr/solr-core/classes/java" />
+    <output-test url="file://$MODULE_DIR$/../../../../idea-build/solr/solr-core/classes/test" />
     <exclude-output />
     <content url="file://$MODULE_DIR$">
       <sourceFolder url="file://$MODULE_DIR$" isTestSource="true" />
Index: dev-tools/idea/solr/core/src/java/solr-core.iml
===================================================================
--- dev-tools/idea/solr/core/src/java/solr-core.iml	(revision 1384534)
+++ dev-tools/idea/solr/core/src/java/solr-core.iml	(working copy)
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <module type="JAVA_MODULE" version="4">
   <component name="NewModuleRootManager" inherit-compiler-output="false">
-    <output url="file://$MODULE_DIR$/../../../build/solr-core/classes/java" />
-    <output-test url="file://$MODULE_DIR$/../../../build/solr-core/classes/test" />
+    <output url="file://$MODULE_DIR$/../../../../idea-build/solr/solr-core/classes/java" />
+    <output-test url="file://$MODULE_DIR$/../../../../idea-build/solr/solr-core/classes/test" />
     <exclude-output />
     <content url="file://$MODULE_DIR$">
       <sourceFolder url="file://$MODULE_DIR$" isTestSource="false" />
Index: dev-tools/idea/solr/solrj/src/test/solrj-tests.iml
===================================================================
--- dev-tools/idea/solr/solrj/src/test/solrj-tests.iml	(revision 1384534)
+++ dev-tools/idea/solr/solrj/src/test/solrj-tests.iml	(working copy)
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <module type="JAVA_MODULE" version="4">
   <component name="NewModuleRootManager" inherit-compiler-output="false">
-    <output url="file://$MODULE_DIR$/../../../build/solr-solrj/classes/java" />
-    <output-test url="file://$MODULE_DIR$/../../../build/solr-solrj/classes/test" />
+    <output url="file://$MODULE_DIR$/../../../../idea-build/solr/solr-solrj/classes/java" />
+    <output-test url="file://$MODULE_DIR$/../../../../idea-build/solr/solr-solrj/classes/test" />
     <exclude-output />
     <content url="file://$MODULE_DIR$">
       <sourceFolder url="file://$MODULE_DIR$" isTestSource="true" />
@@ -16,6 +16,7 @@
     <orderEntry type="module" scope="TEST" module-name="lucene-test-framework" />
     <orderEntry type="module" scope="TEST" module-name="solr-test-framework" />
     <orderEntry type="module" scope="TEST" module-name="solrj" />
+    <orderEntry type="module" scope="TEST" module-name="solrj-test-files" />
     <orderEntry type="module" scope="TEST" module-name="solr-core" />
   </component>
 </module>
Index: dev-tools/idea/solr/solrj/src/test-files/solrj-test-files.iml
===================================================================
--- dev-tools/idea/solr/solrj/src/test-files/solrj-test-files.iml	(revision 1384534)
+++ dev-tools/idea/solr/solrj/src/test-files/solrj-test-files.iml	(working copy)
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <module type="JAVA_MODULE" version="4">
   <component name="NewModuleRootManager" inherit-compiler-output="false">
-    <output url="file://$MODULE_DIR$/../../../build/solr-solrj/classes/java" />
-    <output-test url="file://$MODULE_DIR$/../../../build/solr-solrj/classes/test" />
+    <output url="file://$MODULE_DIR$/../../../../idea-build/solr/solr-solrj/classes/java" />
+    <output-test url="file://$MODULE_DIR$/../../../../idea-build/solr/solr-solrj/classes/test" />
     <exclude-output />
     <content url="file://$MODULE_DIR$">
       <sourceFolder url="file://$MODULE_DIR$" isTestSource="true" />
Index: dev-tools/idea/solr/solrj/src/java/solrj.iml
===================================================================
--- dev-tools/idea/solr/solrj/src/java/solrj.iml	(revision 1384534)
+++ dev-tools/idea/solr/solrj/src/java/solrj.iml	(working copy)
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <module type="JAVA_MODULE" version="4">
   <component name="NewModuleRootManager" inherit-compiler-output="false">
-    <output url="file://$MODULE_DIR$/../../../build/solr-solrj/classes/java" />
-    <output-test url="file://$MODULE_DIR$/../../../build/solr-solrj/classes/test" />
+    <output url="file://$MODULE_DIR$/../../../../idea-build/solr/solr-solrj/classes/java" />
+    <output-test url="file://$MODULE_DIR$/../../../../idea-build/solr/solr-solrj/classes/test" />
     <exclude-output />
     <content url="file://$MODULE_DIR$">
       <sourceFolder url="file://$MODULE_DIR$" isTestSource="false" />
Index: dev-tools/idea/solr/test-framework/solr-test-framework.iml
===================================================================
--- dev-tools/idea/solr/test-framework/solr-test-framework.iml	(revision 1384534)
+++ dev-tools/idea/solr/test-framework/solr-test-framework.iml	(working copy)
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <module type="JAVA_MODULE" version="4">
   <component name="NewModuleRootManager" inherit-compiler-output="false">
-    <output url="file://$MODULE_DIR$/../build/solr-test-framework/classes/java" />
-    <output-test url="file://$MODULE_DIR$/../build/solr-test-framework/classes/test" />
+    <output url="file://$MODULE_DIR$/../../idea-build/solr/solr-test-framework/classes/java" />
+    <output-test url="file://$MODULE_DIR$/../../idea-build/solr/solr-test-framework/classes/test" />
     <exclude-output />
     <content url="file://$MODULE_DIR$">
       <sourceFolder url="file://$MODULE_DIR$/src/java" isTestSource="false" />
Index: dev-tools/idea/solr/contrib/dataimporthandler/dataimporthandler.iml
===================================================================
--- dev-tools/idea/solr/contrib/dataimporthandler/dataimporthandler.iml	(revision 1384534)
+++ dev-tools/idea/solr/contrib/dataimporthandler/dataimporthandler.iml	(working copy)
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <module type="JAVA_MODULE" version="4">
   <component name="NewModuleRootManager" inherit-compiler-output="false">
-    <output url="file://$MODULE_DIR$/../../build/contrib/solr-dataimporthandler/classes/java" />
-    <output-test url="file://$MODULE_DIR$/../../build/contrib/solr-dataimporthandler/classes/test" />
+    <output url="file://$MODULE_DIR$/../../../idea-build/solr/contrib/solr-dataimporthandler/classes/java" />
+    <output-test url="file://$MODULE_DIR$/../../../idea-build/solr/contrib/solr-dataimporthandler/classes/test" />
     <exclude-output />
     <content url="file://$MODULE_DIR$">
       <sourceFolder url="file://$MODULE_DIR$/src/java" isTestSource="false" />
Index: dev-tools/idea/solr/contrib/dataimporthandler-extras/dataimporthandler-extras.iml
===================================================================
--- dev-tools/idea/solr/contrib/dataimporthandler-extras/dataimporthandler-extras.iml	(revision 1384534)
+++ dev-tools/idea/solr/contrib/dataimporthandler-extras/dataimporthandler-extras.iml	(working copy)
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <module type="JAVA_MODULE" version="4">
   <component name="NewModuleRootManager" inherit-compiler-output="false">
-    <output url="file://$MODULE_DIR$/../../build/contrib/solr-dataimporthandler-extras/classes/java" />
-    <output-test url="file://$MODULE_DIR$/../../build/contrib/solr-dataimporthandler-extras/classes/test" />
+    <output url="file://$MODULE_DIR$/../../../idea-build/solr/contrib/solr-dataimporthandler-extras/classes/java" />
+    <output-test url="file://$MODULE_DIR$/../../../idea-build/solr/contrib/solr-dataimporthandler-extras/classes/test" />
     <exclude-output />
     <content url="file://$MODULE_DIR$">
       <sourceFolder url="file://$MODULE_DIR$/src/java" isTestSource="false" />
Index: dev-tools/idea/solr/contrib/analysis-extras/analysis-extras.iml
===================================================================
--- dev-tools/idea/solr/contrib/analysis-extras/analysis-extras.iml	(revision 1384534)
+++ dev-tools/idea/solr/contrib/analysis-extras/analysis-extras.iml	(working copy)
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <module type="JAVA_MODULE" version="4">
   <component name="NewModuleRootManager" inherit-compiler-output="false">
-    <output url="file://$MODULE_DIR$/../../build/contrib/solr-analysis-extras/classes/java" />
-    <output-test url="file://$MODULE_DIR$/../../build/contrib/solr-analysis-extras/classes/test" />
+    <output url="file://$MODULE_DIR$/../../../idea-build/solr/contrib/solr-analysis-extras/classes/java" />
+    <output-test url="file://$MODULE_DIR$/../../../idea-build/solr/contrib/solr-analysis-extras/classes/test" />
     <exclude-output />
     <content url="file://$MODULE_DIR$">
       <sourceFolder url="file://$MODULE_DIR$/src/java" isTestSource="false" />
Index: dev-tools/idea/solr/contrib/uima/uima.iml
===================================================================
--- dev-tools/idea/solr/contrib/uima/uima.iml	(revision 1384534)
+++ dev-tools/idea/solr/contrib/uima/uima.iml	(working copy)
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <module type="JAVA_MODULE" version="4">
   <component name="NewModuleRootManager" inherit-compiler-output="false">
-    <output url="file://$MODULE_DIR$/../../build/contrib/solr-uima/classes/java" />
-    <output-test url="file://$MODULE_DIR$/../../build/contrib/solr-uima/classes/test" />
+    <output url="file://$MODULE_DIR$/../../../idea-build/solr/contrib/solr-uima/classes/java" />
+    <output-test url="file://$MODULE_DIR$/../../../idea-build/solr/contrib/solr-uima/classes/test" />
     <exclude-output />
     <content url="file://$MODULE_DIR$">
       <sourceFolder url="file://$MODULE_DIR$/src/test" isTestSource="true" />
Index: dev-tools/idea/solr/contrib/langid/langid.iml
===================================================================
--- dev-tools/idea/solr/contrib/langid/langid.iml	(revision 1384534)
+++ dev-tools/idea/solr/contrib/langid/langid.iml	(working copy)
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <module type="JAVA_MODULE" version="4">
   <component name="NewModuleRootManager" inherit-compiler-output="false">
-    <output url="file://$MODULE_DIR$/../../build/contrib/solr-langid/classes/java" />
-    <output-test url="file://$MODULE_DIR$/../../build/contrib/solr-langid/classes/test" />
+    <output url="file://$MODULE_DIR$/../../../idea-build/solr/contrib/solr-langid/classes/java" />
+    <output-test url="file://$MODULE_DIR$/../../../idea-build/solr/contrib/solr-langid/classes/test" />
     <exclude-output />
     <content url="file://$MODULE_DIR$">
       <sourceFolder url="file://$MODULE_DIR$/src/test" isTestSource="true" />
Index: dev-tools/idea/solr/contrib/velocity/velocity.iml
===================================================================
--- dev-tools/idea/solr/contrib/velocity/velocity.iml	(revision 1384534)
+++ dev-tools/idea/solr/contrib/velocity/velocity.iml	(working copy)
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <module type="JAVA_MODULE" version="4">
   <component name="NewModuleRootManager" inherit-compiler-output="false">
-    <output url="file://$MODULE_DIR$/../../build/contrib/solr-velocity/classes/java" />
-    <output-test url="file://$MODULE_DIR$/../../build/contrib/solr-velocity/classes/test" />
+    <output url="file://$MODULE_DIR$/../../../idea-build/solr/contrib/solr-velocity/classes/java" />
+    <output-test url="file://$MODULE_DIR$/../../../idea-build/solr/contrib/solr-velocity/classes/test" />
     <exclude-output />
     <content url="file://$MODULE_DIR$">
       <sourceFolder url="file://$MODULE_DIR$/src/test" isTestSource="true" />
Index: dev-tools/idea/solr/contrib/clustering/clustering.iml
===================================================================
--- dev-tools/idea/solr/contrib/clustering/clustering.iml	(revision 1384534)
+++ dev-tools/idea/solr/contrib/clustering/clustering.iml	(working copy)
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <module type="JAVA_MODULE" version="4">
   <component name="NewModuleRootManager" inherit-compiler-output="false">
-    <output url="file://$MODULE_DIR$/../../build/contrib/solr-clustering/classes/java" />
-    <output-test url="file://$MODULE_DIR$/../../build/contrib/solr-clustering/classes/test" />
+    <output url="file://$MODULE_DIR$/../../../idea-build/solr/contrib/solr-clustering/classes/java" />
+    <output-test url="file://$MODULE_DIR$/../../../idea-build/solr/contrib/solr-clustering/classes/test" />
     <exclude-output />
     <content url="file://$MODULE_DIR$">
       <sourceFolder url="file://$MODULE_DIR$/src/java" isTestSource="false" />
Index: dev-tools/idea/solr/contrib/extraction/extraction.iml
===================================================================
--- dev-tools/idea/solr/contrib/extraction/extraction.iml	(revision 1384534)
+++ dev-tools/idea/solr/contrib/extraction/extraction.iml	(working copy)
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <module type="JAVA_MODULE" version="4">
   <component name="NewModuleRootManager" inherit-compiler-output="false">
-    <output url="file://$MODULE_DIR$/../../build/contrib/solr-cell/classes/java" />
-    <output-test url="file://$MODULE_DIR$/../../build/contrib/solr-cell/classes/test" />
+    <output url="file://$MODULE_DIR$/../../../idea-build/solr/contrib/solr-cell/classes/java" />
+    <output-test url="file://$MODULE_DIR$/../../../idea-build/solr/contrib/solr-cell/classes/test" />
     <exclude-output />
     <content url="file://$MODULE_DIR$">
       <sourceFolder url="file://$MODULE_DIR$/src/test" isTestSource="true" />
Index: dev-tools/idea/.idea/workspace.xml
===================================================================
--- dev-tools/idea/.idea/workspace.xml	(revision 1384534)
+++ dev-tools/idea/.idea/workspace.xml	(working copy)
@@ -7,235 +7,231 @@
     <configuration default="false" name="Lucene core" type="JUnit" factoryName="JUnit">
       <module name="lucene-core-tests" />
       <option name="TEST_OBJECT" value="package" />
-      <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$/lucene/build/core" />
+      <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$/idea-build/lucene/core" />
       <option name="VM_PARAMETERS" value="-ea -Dlucene.version=5.0-SNAPSHOT -DtempDir=temp" />
       <option name="TEST_SEARCH_SCOPE"><value defaultName="singleModule" /></option>
     </configuration>
     <configuration default="false" name="Module analyzers-common" type="JUnit" factoryName="JUnit">
       <module name="analysis-common" />
       <option name="TEST_OBJECT" value="package" />
-      <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$/lucene/build/analysis/analyzers-common" />
+      <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$/idea-build/lucene/analysis/common" />
       <option name="VM_PARAMETERS" value="-ea  -DtempDir=temp" />
       <option name="TEST_SEARCH_SCOPE"><value defaultName="singleModule" /></option>
     </configuration>
     <configuration default="false" name="Module analyzers-icu" type="JUnit" factoryName="JUnit">
       <module name="icu" />
       <option name="TEST_OBJECT" value="package" />
-      <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$/lucene/build/analysis/analyzers-icu" />
+      <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$/idea-build/lucene/analysis/icu" />
       <option name="VM_PARAMETERS" value="-ea -DtempDir=temp" />
       <option name="TEST_SEARCH_SCOPE"><value defaultName="singleModule" /></option>
     </configuration>
     <configuration default="false" name="Module analyzers-kuromoji" type="JUnit" factoryName="JUnit">
       <module name="kuromoji" />
       <option name="TEST_OBJECT" value="package" />
-      <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$/lucene/build/analysis/analyzers-kuromoji-idea" />
+      <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$/idea-build/lucene/analysis/kuromoji" />
       <option name="VM_PARAMETERS" value="-ea -DtempDir=temp" />
       <option name="TEST_SEARCH_SCOPE"><value defaultName="singleModule" /></option>
     </configuration>
     <configuration default="false" name="Module analyzers-morfologik" type="JUnit" factoryName="JUnit">
       <module name="morfologik" />
       <option name="TEST_OBJECT" value="package" />
-      <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$/lucene/build/analysis/analyzers-morfologik" />
+      <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$/idea-build/lucene/analysis/morfologik" />
       <option name="VM_PARAMETERS" value="-ea -DtempDir=temp" />
       <option name="TEST_SEARCH_SCOPE"><value defaultName="singleModule" /></option>
     </configuration>
     <configuration default="false" name="Module analyzers-phonetic" type="JUnit" factoryName="JUnit">
       <module name="phonetic" />
       <option name="TEST_OBJECT" value="package" />
-      <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$/lucene/build/analysis/analyzers-phonetic" />
+      <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$/idea-build/lucene/analysis/phonetic" />
       <option name="VM_PARAMETERS" value="-ea -DtempDir=temp" />
       <option name="TEST_SEARCH_SCOPE"><value defaultName="singleModule" /></option>
     </configuration>
     <configuration default="false" name="Module analyzers-smartcn" type="JUnit" factoryName="JUnit">
       <module name="smartcn" />
       <option name="TEST_OBJECT" value="package" />
-      <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$/lucene/build/analysis/analyzers-smartcn" />
+      <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$/idea-build/lucene/analysis/smartcn" />
       <option name="VM_PARAMETERS" value="-ea -DtempDir=temp" />
       <option name="TEST_SEARCH_SCOPE"><value defaultName="singleModule" /></option>
     </configuration>
     <configuration default="false" name="Module analyzers-stempel" type="JUnit" factoryName="JUnit">
       <module name="stempel" />
       <option name="TEST_OBJECT" value="package" />
-      <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$/lucene/build/analysis/analyzers-stempel" />
+      <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$/idea-build/lucene/analysis/stempel" />
       <option name="VM_PARAMETERS" value="-ea -DtempDir=temp" />
       <option name="TEST_SEARCH_SCOPE"><value defaultName="singleModule" /></option>
     </configuration>
     <configuration default="false" name="Module analyzers-uima" type="JUnit" factoryName="JUnit">
       <module name="analysis-uima" />
       <option name="TEST_OBJECT" value="package" />
-      <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$/lucene/build/analysis/analyzers-uima" />
+      <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$/idea-build/lucene/analysis/uima" />
       <option name="VM_PARAMETERS" value="-ea -DtempDir=temp" />
       <option name="TEST_SEARCH_SCOPE"><value defaultName="singleModule" /></option>
     </configuration>
     <configuration default="false" name="Module benchmark" type="JUnit" factoryName="JUnit">
       <module name="benchmark" />
       <option name="TEST_OBJECT" value="package" />
-      <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$/lucene/build/benchmark" />
+      <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$/idea-build/lucene/benchmark" />
       <option name="VM_PARAMETERS" value="-ea -DtempDir=temp" />
       <option name="TEST_SEARCH_SCOPE"><value defaultName="singleModule" /></option>
-      <method>
-        <option name="AntTarget" enabled="true" target="copy-alg-files-for-testing"
-                antfile="file://$PROJECT_DIR$/lucene/benchmark/build.xml" />
-      </method>
     </configuration>
     <configuration default="false" name="Module classification" type="JUnit" factoryName="JUnit">
       <module name="classification" />
       <option name="TEST_OBJECT" value="package" />
-      <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$/lucene/build/classification" />
+      <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$/idea-build/lucene/classification" />
       <option name="VM_PARAMETERS" value="-ea -DtempDir=temp" />
       <option name="TEST_SEARCH_SCOPE"><value defaultName="singleModule" /></option>
     </configuration>
     <configuration default="false" name="Module codecs" type="JUnit" factoryName="JUnit">
       <module name="codecs-tests" />
       <option name="TEST_OBJECT" value="package" />
-      <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$/lucene/build/codecs" />
+      <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$/idea-build/lucene/codecs" />
       <option name="VM_PARAMETERS" value="-ea -DtempDir=temp" />
       <option name="TEST_SEARCH_SCOPE"><value defaultName="singleModule" /></option>
     </configuration>
     <configuration default="false" name="Module facet" type="JUnit" factoryName="JUnit">
       <module name="facet" />
       <option name="TEST_OBJECT" value="package" />
-      <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$/lucene/build/facet" />
+      <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$/idea-build/lucene/facet" />
       <option name="VM_PARAMETERS" value="-ea -DtempDir=temp" />
       <option name="TEST_SEARCH_SCOPE"><value defaultName="singleModule" /></option>
     </configuration>
     <configuration default="false" name="Module grouping" type="JUnit" factoryName="JUnit">
       <module name="grouping" />
       <option name="TEST_OBJECT" value="package" />
-      <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$/lucene/build/grouping" />
+      <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$/idea-build/lucene/grouping" />
       <option name="VM_PARAMETERS" value="-ea -DtempDir=temp" />
       <option name="TEST_SEARCH_SCOPE"><value defaultName="singleModule" /></option>
     </configuration>
     <configuration default="false" name="Module highlighter" type="JUnit" factoryName="JUnit">
       <module name="highlighter" />
       <option name="TEST_OBJECT" value="package" />
-      <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$/lucene/build/highlighter" />
+      <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$/idea-build/lucene/highlighter" />
       <option name="VM_PARAMETERS" value="-ea -DtempDir=temp" />
       <option name="TEST_SEARCH_SCOPE"><value defaultName="singleModule" /></option>
     </configuration>
     <configuration default="false" name="Module join" type="JUnit" factoryName="JUnit">
       <module name="join" />
       <option name="TEST_OBJECT" value="package" />
-      <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$/lucene/build/join" />
+      <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$/idea-build/lucene/join" />
       <option name="VM_PARAMETERS" value="-ea -DtempDir=temp" />
       <option name="TEST_SEARCH_SCOPE"><value defaultName="singleModule" /></option>
     </configuration>
     <configuration default="false" name="Module memory" type="JUnit" factoryName="JUnit">
       <module name="memory" />
       <option name="TEST_OBJECT" value="package" />
-      <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$/lucene/build/memory" />
+      <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$/idea-build/lucene/memory" />
       <option name="VM_PARAMETERS" value="-ea -DtempDir=temp" />
       <option name="TEST_SEARCH_SCOPE"><value defaultName="singleModule" /></option>
     </configuration>
     <configuration default="false" name="Module misc" type="JUnit" factoryName="JUnit">
       <module name="misc" />
       <option name="TEST_OBJECT" value="package" />
-      <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$/lucene/build/misc" />
+      <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$/idea-build/lucene/misc" />
       <option name="VM_PARAMETERS" value="-ea -DtempDir=temp" />
       <option name="TEST_SEARCH_SCOPE"><value defaultName="singleModule" /></option>
     </configuration>
     <configuration default="false" name="Module queries" type="JUnit" factoryName="JUnit">
       <module name="queries" />
       <option name="TEST_OBJECT" value="package" />
-      <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$/lucene/build/queries" />
+      <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$/idea-build/lucene/queries" />
       <option name="VM_PARAMETERS" value="-ea -DtempDir=temp" />
       <option name="TEST_SEARCH_SCOPE"><value defaultName="singleModule" /></option>
     </configuration>
     <configuration default="false" name="Module queryparser" type="JUnit" factoryName="JUnit">
       <module name="queryparser" />
       <option name="TEST_OBJECT" value="package" />
-      <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$/lucene/build/queries" />
+      <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$/idea-build/lucene/queries" />
       <option name="VM_PARAMETERS" value="-ea -DtempDir=temp" />
       <option name="TEST_SEARCH_SCOPE"><value defaultName="singleModule" /></option>
     </configuration>
     <configuration default="false" name="Module sandbox" type="JUnit" factoryName="JUnit">
       <module name="sandbox" />
       <option name="TEST_OBJECT" value="package" />
-      <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$/lucene/build/sandbox" />
+      <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$/idea-build/lucene/sandbox" />
       <option name="VM_PARAMETERS" value="-ea -DtempDir=temp" />
       <option name="TEST_SEARCH_SCOPE"><value defaultName="singleModule" /></option>
     </configuration>
     <configuration default="false" name="Module spatial" type="JUnit" factoryName="JUnit">
       <module name="spatial" />
       <option name="TEST_OBJECT" value="package" />
-      <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$/lucene/build/spatial" />
+      <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$/idea-build/lucene/spatial" />
       <option name="VM_PARAMETERS" value="-ea -DtempDir=temp" />
       <option name="TEST_SEARCH_SCOPE"><value defaultName="singleModule" /></option>
     </configuration>
     <configuration default="false" name="Module suggest" type="JUnit" factoryName="JUnit">
       <module name="suggest" />
       <option name="TEST_OBJECT" value="package" />
-      <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$/lucene/build/suggest" />
+      <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$/idea-build/lucene/suggest" />
       <option name="VM_PARAMETERS" value="-ea -DtempDir=temp" />
       <option name="TEST_SEARCH_SCOPE"><value defaultName="singleModule" /></option>
     </configuration>
     <configuration default="false" name="Solr core" type="JUnit" factoryName="JUnit">
       <module name="solr-core-tests" />
       <option name="TEST_OBJECT" value="package" />
-      <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$/solr/build/solr-core" />
+      <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$/idea-build/solr/solr-core" />
       <option name="VM_PARAMETERS" value="-ea -Dtests.luceneMatchVersion=5.0 -DtempDir=temp -Djetty.testMode=1 -Djetty.insecurerandom=1 -Dsolr.directoryFactory=org.apache.solr.core.MockDirectoryFactory" />
       <option name="TEST_SEARCH_SCOPE"><value defaultName="singleModule" /></option>
     </configuration>
     <configuration default="false" name="Solrj" type="JUnit" factoryName="JUnit">
       <module name="solrj-tests" />
       <option name="TEST_OBJECT" value="package" />
-      <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$/solr/build/solr-solrj" />
+      <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$/idea-build/solr/solr-solrj" />
       <option name="VM_PARAMETERS" value="-ea -Dtests.luceneMatchVersion=5.0 -DtempDir=temp -Djetty.testMode=1 -Djetty.insecurerandom=1 -Dsolr.directoryFactory=org.apache.solr.core.MockDirectoryFactory" />
       <option name="TEST_SEARCH_SCOPE"><value defaultName="singleModule" /></option>
     </configuration>
     <configuration default="false" name="Solr analysis-extras contrib" type="JUnit" factoryName="JUnit">
       <module name="analysis-extras" />
       <option name="TEST_OBJECT" value="package" />
-      <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$/solr/build/contrib/solr-analysis-extras" />
+      <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$/idea-build/solr/contrib/solr-analysis-extras" />
       <option name="VM_PARAMETERS" value="-ea -Dtests.luceneMatchVersion=5.0 -DtempDir=temp -Djetty.testMode=1 -Djetty.insecurerandom=1 -Dsolr.directoryFactory=org.apache.solr.core.MockDirectoryFactory" />
       <option name="TEST_SEARCH_SCOPE"><value defaultName="singleModule" /></option>
     </configuration>
     <configuration default="false" name="Solr clustering contrib" type="JUnit" factoryName="JUnit">
       <module name="clustering" />
       <option name="TEST_OBJECT" value="package" />
-      <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$/solr/build/contrib/solr-clustering" />
+      <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$/idea-build/solr/contrib/solr-clustering" />
       <option name="VM_PARAMETERS" value="-ea -Dtests.luceneMatchVersion=5.0 -DtempDir=temp -Djetty.testMode=1 -Djetty.insecurerandom=1 -Dsolr.directoryFactory=org.apache.solr.core.MockDirectoryFactory" />
       <option name="TEST_SEARCH_SCOPE"><value defaultName="singleModule" /></option>
     </configuration>
     <configuration default="false" name="Solr dataimporthandler contrib" type="JUnit" factoryName="JUnit">
       <module name="dataimporthandler" />
       <option name="TEST_OBJECT" value="package" />
-      <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$/solr/build/contrib/solr-dataimporthandler" />
+      <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$/idea-build/solr/contrib/solr-dataimporthandler" />
       <option name="VM_PARAMETERS" value="-ea -DtempDir=temp -Djetty.testMode=1 -Djetty.insecurerandom=1 -Dsolr.directoryFactory=org.apache.solr.core.MockDirectoryFactory" />
       <option name="TEST_SEARCH_SCOPE"><value defaultName="singleModule" /></option>
     </configuration>
     <configuration default="false" name="Solr dataimporthandler-extras contrib" type="JUnit" factoryName="JUnit">
       <module name="dataimporthandler-extras" />
       <option name="TEST_OBJECT" value="package" />
-      <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$/solr/build/contrib/solr-dataimporthandler-extras" />
+      <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$/idea-build/solr/contrib/solr-dataimporthandler-extras" />
       <option name="VM_PARAMETERS" value="-ea -DtempDir=temp -Djetty.testMode=1 -Djetty.insecurerandom=1 -Dsolr.directoryFactory=org.apache.solr.core.MockDirectoryFactory" />
       <option name="TEST_SEARCH_SCOPE"><value defaultName="singleModule" /></option>
     </configuration>
     <configuration default="false" name="Solr extraction contrib" type="JUnit" factoryName="JUnit">
       <module name="extraction" />
       <option name="TEST_OBJECT" value="package" />
-      <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$/solr/build/contrib/solr-cell" />
+      <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$/idea-build/solr/contrib/solr-cell" />
       <option name="VM_PARAMETERS" value="-ea -DtempDir=temp -Djetty.testMode=1 -Djetty.insecurerandom=1 -Dsolr.directoryFactory=org.apache.solr.core.MockDirectoryFactory" />
       <option name="TEST_SEARCH_SCOPE"><value defaultName="singleModule" /></option>
     </configuration>
     <configuration default="false" name="Solr langid contrib" type="JUnit" factoryName="JUnit">
       <module name="langid" />
       <option name="TEST_OBJECT" value="package" />
-      <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$/solr/build/contrib/solr-langid" />
+      <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$/idea-build/solr/contrib/solr-langid" />
       <option name="VM_PARAMETERS" value="-ea -DtempDir=temp -Djetty.testMode=1 -Djetty.insecurerandom=1 -Dsolr.directoryFactory=org.apache.solr.core.MockDirectoryFactory" />
       <option name="TEST_SEARCH_SCOPE"><value defaultName="singleModule" /></option>
     </configuration>
     <configuration default="false" name="Solr uima contrib" type="JUnit" factoryName="JUnit">
       <module name="uima" />
       <option name="TEST_OBJECT" value="package" />
-      <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$/solr/build/contrib/solr-uima" />
+      <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$/idea-build/solr/contrib/solr-uima" />
       <option name="VM_PARAMETERS" value="-ea -Dtests.luceneMatchVersion=5.0 -DtempDir=temp -Djetty.testMode=1 -Djetty.insecurerandom=1 -Dsolr.directoryFactory=org.apache.solr.core.MockDirectoryFactory" />
       <option name="TEST_SEARCH_SCOPE"><value defaultName="singleModule" /></option>
     </configuration>
     <configuration default="false" name="Solr velocity contrib" type="JUnit" factoryName="JUnit">
       <module name="velocity" />
       <option name="TEST_OBJECT" value="package" />
-      <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$/solr/build/contrib/solr-velocity" />
+      <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$/idea-build/solr/contrib/solr-velocity" />
       <option name="VM_PARAMETERS" value="-ea" />
       <option name="TEST_SEARCH_SCOPE"><value defaultName="singleModule" /></option>
     </configuration>
Index: dev-tools/idea/.idea/modules.xml
===================================================================
--- dev-tools/idea/.idea/modules.xml	(revision 1384534)
+++ dev-tools/idea/.idea/modules.xml	(working copy)
@@ -16,7 +16,8 @@
       <module filepath="$PROJECT_DIR$/lucene/analysis/smartcn/smartcn.iml" />
       <module filepath="$PROJECT_DIR$/lucene/analysis/stempel/stempel.iml" />
       <module filepath="$PROJECT_DIR$/lucene/analysis/uima/analysis-uima.iml" />
-      <module filepath="$PROJECT_DIR$/lucene/benchmark/benchmark.iml" />
+      <module filepath="$PROJECT_DIR$/lucene/benchmark/src/benchmark.iml" />
+      <module filepath="$PROJECT_DIR$/lucene/benchmark/conf/benchmark-conf.iml" />
       <module filepath="$PROJECT_DIR$/lucene/classification/classification.iml" />
       <module filepath="$PROJECT_DIR$/lucene/codecs/src/java/codecs.iml" />
       <module filepath="$PROJECT_DIR$/lucene/codecs/src/test/codecs-tests.iml" />
Index: build.xml
===================================================================
--- build.xml	(revision 1384534)
+++ build.xml	(working copy)
@@ -202,6 +202,7 @@
       <fileset dir="solr" includes="**/*.iml"/>
       <fileset dir="lucene" includes="**/*.iml"/>
     </delete>
+    <delete dir="idea-build" failonerror="true"/>
   </target>
 
   <target name="clean" description="Clean Lucene and Solr">
